main.cpp

```diff
@@ -51,86 +51,86 @@ arm.flex();                             // 3)
 arm.rotateCounterClockwise();           // 4) demonstrates CamelCase
 
 
-//  1)
+//  speaker //  1)
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  2)
+//  action 1: the speaker blasts
+speaker.blastMusic();
+//  action 2: the speaker powers off
+speaker.powerOff();
+//  action 3: the speaker changes volume
+speaker.changeVolume();
+//  fan. 2)
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  3)
+//  action 1: the fan turns on
+fan.on();
+//  action 2: the fan turns off
+fan.off();
+//  action 3: the fan rotates
+fan.rotate();
+//  monitor  3)
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  4)
+//  action 1: the monitor selects HDMI
+monitor.selectHDMI();
+//  action 2: the monitor selects Bluetooth
+monitor.selectBluetooth();
+//  action 3: the monitor selects USBC
+monitor.selectUSBC();
+//  cat  4)
 //  Noun:
+//  action 1: the cat runs
+cat.run();
+//  action 2: the cat jumps
+cat.jump();
+//  action 3: the cat sleeps
+cat.sleep();
+//  dog  5)
+//  Noun: the dog barks
 //  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  5)
+dog.bark();
+//  action 2: the dog digs
+dog.dig();
+//  action 3: the dog eats
+dog.eat();
+//  tree  6)
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  6)
+//  action 1: the tree grows
+tree.grow();
+//  action 2: the tree branches
+tree.branch();
+//  action 3: the tree photosynthesizes
+tree.photosynthesize();
+//  taco truck  7)
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  7)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  8)
+//  action 1: the taco truck makes tacos
+tacoTruck.makeTaco();
+//  action 2: the taco truck drives
+tacoTruck.drive();
+//  action 3: the taco truck makes change
+tacoTruck.makeChange();
+//  heater  8)
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  action 1: the heater sets to high
+heater.high();
+//  action 2: the heater sets to medium
+heater.medium();
+//  action 3: the heater sets to low
+heater.low();
 //  9)
+//  Noun: air purifier
+//  action 1: the air purifier turns on
+airPurifier.on();
+//  action 2: the air purifier turns off
+airPurifier.off();
+//  action 3: the air purifier changes to the next speed
+airPurifier.nextSpeed();
+//   bird 10)
 //  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
-//  10)
-//  Noun:
-//  action 1:
-
-//  action 2:
-
-//  action 3:
-
+//  action 1: the bird chirps
+bird.chirp();
+//  action 2: the bird flys
+bird.fly();
+//  action 3: the bird perches
+bird.perch();

```

Comment:
As explained in the instructions, here is the obligatory request for some changes to get you familiar with the Pull Request Review process.
REPLACE the highlighted noun/action as follows:
- [ ] come up with a new noun and 3 actions
- [ ] pick a noun that needs `camelCase` for the name
- [ ] pick actions that need `camelCase` for their name.

Do not create an 11th noun.

Once you have completed the above-requested changes, please do the following:
- click each checkbox to mark each requested change as completed.
- click the `[Resolve Conversation]` button.
- Re-request a review of your pull request. It's on the right side of the Pull request page where you originally requested my review.
- Send me the link to your pull request in a new message.